### ISSUE TITLE
TypedDict does not support calling function on output of `dict.get()`

### ISSUE THREAD

**Issue by bfelbo** [microsoft/pyright#425]
**Describe the bug**
Pyright raises an issue when calling a function on the output of the dict.get() function on an TypedDict. This is despite knowing types when both in the case when the key is set (from TypedDict definition) and when it's not (i.e. second argument of get()). The code runs fine.

In this example, pyright raises:
```
Argument of type 'Union[object, Any]' cannot be assigned to parameter '__o' of type 'Sized'
  Type 'object' cannot be assigned to type 'Sized'
    '__len__' is not present
```

**To Reproduce**
Run pyright on the code below.

**Expected behavior**
No error.

**Screenshots or Code**
```
from typing_extensions import TypedDict
UserType = TypedDict("UserType", {"bio": str, "name": str}, total=False)
user: UserType = {"bio": "Sup.", "name": "Bob"}
print(len(user.get("bio", "")))
```

**VS Code extension or command-line**
VS Code extension 1.1.10

**Comment by erictraut** [comment]
Thanks for the suggestion. This will be included in the next release.

**Comment by erictraut** [comment]
This is now fixed in 1.1.14, which I just published.
